parser/src/moves.rs

```diff
@@ -165,6 +165,13 @@ pub(crate) fn aerated<M: Move + Copy>(
     blanks().then(m).and_then(blanks())
 }
 
+pub(crate) fn identifier_parenthesis() -> AndThenMove<
+    PredicateMove<impl Fn(Token) -> bool + Copy + Sized>,
+    PredicateMove<impl Fn(Token) -> bool + Copy + Sized>,
+> {
+    of_type(Identifier).and_then(of_types(&[RoundedLeftBracket, SquaredLeftBracket]))
+}
+
```

Comment:
You should move this under the `STANDARD MOVES` section of the same file
https://github.com/moshell-lang/moshell/pull/85/files#diff-39df3f30535471a9f8520ed353ee96182c28fe313a757245b6082f595608c77fR352